src/UserSettingsPage.cpp

```diff
@@ -941,6 +958,12 @@ UserSettingsPage::UserSettingsPage(QSharedPointer<UserSettings> settings, QWidge
         boxWrap(tr("Circular Avatars"),
                 avatarCircles_,
                 tr("Change the appearance of user avatars in chats.\nOFF - square, ON - Circle."));
+        if (JdenticonProvider::isAvailable())
```

Comment:
I think this button should always be there, just disabled (so not toggleable), if the plugin is not available.

Reply:
Sure, I can do that